resources/content/schema/base/cluster.json

```diff
@@ -0,0 +1,35 @@
+{
+    "resourceFields": {
+        "discoverySpec": {
+            "type": "string",
+            "create": true,
+            "update": true,
+            "nullable": true,
+            "description": "Equivalent to the value used for the --discovery parameter for the cluster server"
+        },
+        "port": {
+            "type": "int",
+            "create": true,
+            "update": true,
+            "nullable": false,
+            "description":"Port the cluster server is running on"
+        },
+        "certificateDataRef": {
```

Comment:
Although, not sure whether I'll actually need to do a join.  I think for now, I'm able to get away with just making 2 queries.